nasl/nasl_misc_funcs.c

```diff
@@ -270,8 +272,10 @@ nasl_end_denial (lex_ctxt *lexic)
       if (soc > 0)
         {
           /* Send some data */
-#define BOGUS "are you dead ?"
-          if ((nsend (soc, BOGUS, sizeof (BOGUS) - 1, 0)) >= 0)
+          snprintf (bogus_data, sizeof (bogus_data),
+                    "Network Security Scan In Progress run by %s",
```

Comment:
```suggestion
                    "Network Security Scan by %s in progress.",
```

Not absolutely sure but that might sound a little bit better.